frontend/src/routes/Credentials/CreateCredentialsCatalog.tsx

```diff
@@ -44,10 +44,25 @@ export function CreateCredentialsCatalog() {
             provider === Provider.aws
               ? nextStep(NavigationPath.addAWSType)
               : nextStep(getTypedCreateCredentialsPath(provider)),
+          ...(provider === Provider.redhatvirtualization
+            ? ({
+                alertTitle: t('Deprecated host platform'),
+                alertVariant: 'warning',
+                alertContent: (
+                  <>
+                    {t(
+                      'Red Hat Virtualization is deprecated as a host platform for OpenShift 4.13 and will be removed in the next release.'
+                    )}
+                    <br />
+                    <ViewDocumentationLink doclink={DOC_LINKS.RHV_DEPRECATION} />
+                  </>
+                ),
+              } as ICatalogCard)
```

Comment:
Simplest fix for this TypeScript issue is to change line 50 to `alertVariant: 'warning' as const,` or you could also change this line to `} as Partial<ICatalogCard>)`... it's not really an issue in this case, but in general we should avoid casting to a certain type. An `ICatalogCard` must have `id` and `title` props, which this object does not.

Reply:
Thank you, I struggled with this one longer than I care to admit.  Updating.